static/app/views/issueList/pages/dynamicGrouping.tsx

```diff
@@ -14,132 +14,95 @@ import EventOrGroupTitle from 'sentry/components/eventOrGroupTitle';
 import EventMessage from 'sentry/components/events/eventMessage';
 import TimesTag from 'sentry/components/group/inboxBadges/timesTag';
 import UnhandledTag from 'sentry/components/group/inboxBadges/unhandledTag';
-import IssueReplayCount from 'sentry/components/group/issueReplayCount';
 import ProjectBadge from 'sentry/components/idBadge/projectBadge';
-import Placeholder from 'sentry/components/placeholder';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
 import Redirect from 'sentry/components/redirect';
-import {IconChat, IconStar} from 'sentry/icons';
 import {t, tn} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {Group} from 'sentry/types/group';
-import {defined} from 'sentry/utils';
 import {getMessage, getTitle} from 'sentry/utils/events';
 import {useApiQuery} from 'sentry/utils/queryClient';
-import useReplayCountForIssues from 'sentry/utils/replayCount/useReplayCountForIssues';
-import {projectCanLinkToReplay} from 'sentry/utils/replays/projectSupportsReplay';
 import useOrganization from 'sentry/utils/useOrganization';
 import {useUser} from 'sentry/utils/useUser';
 import {useUserTeams} from 'sentry/utils/useUserTeams';
 
 interface AssignedEntity {
-  email: string | null;
+  email: string | null; // unused
   id: string;
-  name: string;
+  name: string; // unused
   type: string;
 }
 
 interface ClusterSummary {
   assignedTo: AssignedEntity[];
-  cluster_avg_similarity: number | null;
+  cluster_avg_similarity: number | null; // unused
   cluster_id: number;
-  cluster_min_similarity: number | null;
-  cluster_size: number | null;
+  cluster_min_similarity: number | null; // unused
+  cluster_size: number | null; // unused
   description: string;
   fixability_score: number | null;
   group_ids: number[];
-  issue_titles: string[];
-  project_ids: number[];
-  tags: string[];
+  issue_titles: string[]; // unused
+  project_ids: number[]; // unused
+  tags: string[]; // unused
```

Comment:
are these left in intentionally?

Reply:
for now, i'm keeping them in the data json thingy so i don't want to remove them from the schema right now